stats/kernel-selftests

```diff
@@ -1112,37 +1112,50 @@ class NetfilterStater < Stater
       @test_case = $1 if @test_script == 'nft_concat_range.sh'
     when /^#   (.+)\[( OK|FAIL|SKIP)/
       #   Add two elements, flush, re-add                               [ OK ]
-      @subtest_case = $1.strip if @test_script == 'nft_concat_range.sh'
-      stats.add "#{@test_prefix}.#{@test_case}.#{$1}", $2 if @test_script == 'nft_concat_range.sh'
+      test_case = $1
+      result = $2
+      parse_subtest_result(test_case, result, stats)
```

Comment:
thanks, it's ok here to just call arse_subtest_result($1, $2, stats) without earlier 2 assignments